x/group/internal/math/dec.go

```diff
@@ -6,7 +6,7 @@ import (
 
 	"github.com/cockroachdb/apd/v2"
 
-	"github.com/cosmos/cosmos-sdk/types/errors"
+	"github.com/cosmos/cosmos-sdk/x/group/errors"
```

Comment:
`x/group/errors` doesn't have `wrap` method defined in it which  is used in L41

Reply:
good catch, fixed!